resources/assets/js/Widgets/Submissions.js

```diff
@@ -35,4 +58,11 @@ export default class Submissions extends Component {
       </div>
     );
   }
+
+}
+
+
+
+if (document.getElementById('submission-widget')) {
+  ReactDOM.render(<Submissions/>, document.getElementById('submission-widget'));
 }
```

Comment:
Don't think we need lines 66-68 anymore as the Submissions component is being imported by other components.